libs/components/tabs/src/lib/modules/sectioned-form/sectioned-form.component.scss

```diff
@@ -47,10 +47,60 @@
   }
 }
 
+:host-context(sky-modal) {
+  .sky-sectioned-form {
+    min-height: 460px;
+  }
+}
+
+:host-context(sky-modal-content) {
+  :host .sky-sectioned-form {
+    margin: var(--sky-compat-sectioned-form-modal-margin);
+  }
+
+  form :host .sky-sectioned-form {
+    margin: var(--sky-compat-sectioned-form-modal-margin);
+  }
+}
+
+:host-context(sky-modal-content) > {
+  :host .sky-sectioned-form {
+    margin: -$sky-margin-plus-half;
+  }
+
+  form :host .sky-sectioned-form {
+    margin: -$sky-margin-plus-half;
+  }
+}
+
 .sky-sectioned-form-tabs-hidden {
   display: none;
 }
 
 .sky-sectioned-form-content-hidden {
   display: none;
 }
+
+@include mixins.sky-theme-modern {
+  :host-context(sky-modal-content) {
+    :host .sky-sectioned-form {
+      margin: var(--sky-compat-sectioned-form-theme-modern-modal-margin);
+    }
+
+    form :host .sky-sectioned-form {
+      margin: var(--sky-compat-sectioned-form-theme-modern-modal-margin);
+    }
+  }
+
+  :host-context(sky-modal-content) > {
```

Comment:
Be sure to test this in Firefox. I seem to recall an issue with using `:host-context` in a mixin where `:host-context` is preserved in the resulting CSS and breaks in Firefox.